lib/send_sms.rb

```diff
@@ -0,0 +1,16 @@
+require 'twilio-ruby'
+
+time = Time.new + 3600 
```

Comment:
nice saving this as a variable - didn't think of this myself but much easier to read